Create file-paths.md (#766)
Codecademy/docs: content/general/concepts/file-paths/file-paths.md

---
Title: 'File Paths'

> Description: 'A file path describes the location of a file in a web site's folder structure. They are used to link to external files.'

Reviewer: ```suggestion
Description: 'A file path describes the location of a file in a computer's folder structure.'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'File Paths'
  - 'Files'
  - 'URL'
CatalogContent:
  - 'learn-html'
  - 'paths/computer-science'
---

A file path describes the location of a file in a computer's folder structure. They are used to locate files, like:

- Images
- Other programs

> - Libretys/frame works

Reviewer: ```suggestion
- Libraries/frameworks
```

---
Title: 'File Paths'

> Description: 'A file path describes the location of a file in a web sites folder structure. They are used to link to external files.'

Reviewer: ```suggestion
Description: 'A file path describes the location of a file in a web site's folder structure. They are used to link to external files.'
```

Reviewer: Also: the general subject of file paths are important to the file structure on any computer, not just web applications.
So something like this would be better: "A file path describes the location of a file in a computer's folder structure."

Subjects:
  - 'Web Development'

>   - 'Web Design'

Reviewer: ```suggestion
  - 'Computer Science'
```

Tags:
  - 'File Paths'
  - 'Files'
  - 'URL'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> A file path describes the location of a file in a programs folder structure. They are used to link files, like:

Reviewer: ```suggestion
A file path describes the location of a file in a computer's folder structure. They are used to locate files, like:
```

- Websites
- Images
- Videos
- Other programs

## Absolute File Paths

> URL paths in HTML can be absolute paths, like a full URL, for example:

Reviewer: A URL is not the same as a file path. A web server can change the URL to point to a completely different file path on the local machine.  I think this should be talking primarily about file paths on the local machine, then discuss the relationship to URLs and relative paths. It should focus as much or more on using them to navigate a computer's file system as it does using them in a web context.

- `./js/script.js`
- `./scripts/main.py`

> Relative file paths begin with `./` followed by a path to the local file. `./` tells the program to look for the file path from the current folder. To go up 1 level im the file structer you use `../`.

Reviewer 2: ```suggestion
Relative file paths begin with `./` followed by a path to the local file. `./` tells the program to look for the file path from the current folder. To go up one level in the file structure, two periods followed by a forward-slash are used: `../`.
```
